client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -50,10 +50,18 @@ class StatsPeriodNavigation extends PureComponent {
 		}
 	};
 
+	isHoursPeriod = ( period ) => 'hour' === period;
+
+	getNumberOfDays = ( isEmailStats, period, maxBars ) =>
+		isEmailStats && ! this.isHoursPeriod( period ) ? maxBars : 1;
```

Comment:
I don't think it's necesarry to pass `maxBars `around, since it's already in `this.props`. We might want to refactor this later.

Reply:
I like pure functions :D